__tests__/app.test.js

```diff
@@ -2,14 +2,100 @@ const pool = require('../lib/utils/pool');
 const setup = require('../data/setup');
 const request = require('supertest');
 const app = require('../lib/app');
+const Book = require('../lib/models/Book');
+const Author = require('../lib/models/Author');
 
 describe('backend-express-template routes', () => {
   beforeEach(() => {
     return setup(pool);
   });
-  it('example test - delete me!', () => {
-    expect(1).toEqual(1);
+  it('GET book route returns all books in the data set', async () => {
+    const res = await request(app).get('/books');
+    const book = res.body.find((book) => book.id === '1');
+    expect(book).toHaveProperty('title', 'Slaughterhouse Five');
   });
+
+  it('GET single book with author data', async () => {
+    const res = await request(app).get('/books/1');
+    const expected = {
+      'id': '1',
+      'title': 'Slaughterhouse Five',
+      'publisher': 'Delacorte',
+      'released': 1969,
+      'authors': [
+        {
+          'id': 1,
+          'dob': 'November 11, 1922',
+          'pob': 'Indiana, US',
+          'name': 'Kurt Vonnegut'
+        }
+      ]
+    };
+    expect(res.body).toEqual(expected);
+  });
+
+  it('GET author route returns all authors with their respective books', async () => {
+    const res = await request(app).get('/authors');
+    const author = res.body.find((author) => author.id === '2');
+    expect(author).toHaveProperty('name', 'Neil Gaiman');
+  });
+
```

Comment:
for your next assignment, split the tests into different files based on table. 
if you were to refactor this one, those files would be `authors.test.js` and `books.test.js`